lib/suspenders/generators/stylesheet_base_generator.rb

```diff
@@ -0,0 +1,23 @@
+require "rails/generators"
+
+module Suspenders
+  class StylesheetBaseGenerator < Rails::Generators::Base
+    source_root File.expand_path(
+      File.join("..", "..", "..", "templates"),
+      File.dirname(__FILE__))
+
+    def add_stylesheet_gems
+      gem "bourbon", "5.0.0.beta.5"
+      gem "neat", "~> 1.7.0"
+      gem "refills", group: [:development, :test]
+    end
+
+    def add_css_config
+      copy_file(
+        "application.scss",
+        "app/assets/stylesheets/application.scss",
+        force: true,
+      )
```

Comment:
> I mean: leave this line as-is, but add an additional check for app/assets/application.css and if it exists print a warning.

If we change and move the original file, you see the changes in the git diff and act accordingly. If we add the new file, it might come as a surprise that both files exist, and potentially be more confusing, which makes me prefer modifying the existing file.

If you'd rather add the warning message, we can keep this in mind, and if people trip on this we can later improve. Your call.